src/olympia/reviewers/views.py

```diff
@@ -1250,7 +1251,9 @@ def set_needs_human_review(self, request, **kwargs):
             Version, pk=request.data.get('version'), addon_id=kwargs['pk']
         )
         status_code = status.HTTP_202_ACCEPTED
-        version.update(needs_human_review=True)
+        NeedsHumanReview.objects.create(
+            version=version, reason=NeedsHumanReview.REASON_MANUALLY_SET_BY_REVIEWER
+        )
```

Comment:
After thinking a bit more about this: I think in this case we don't care, and we want a new flag, since it's a manual very explicit operation by a reviewer.